WriteHDLUtils.py

```diff
@@ -790,6 +846,13 @@ def writeModuleInst_generic(module, hls_src_dir, f_writeTemplatePars,
             if first_of_type:
                 string_bx_out += writeProcBXPort(module.mtype_short(),False,False) # output bx
         else:
+            if "table" in argname:
```

Comment:
Could this be moved to a separate elif before the else? There's no need looping over the memory modules to find a match for the LUTs I assume.

Reply:
Thanks Mei-Li! Great point. Fixed.